marklogic-data-hub/src/main/java/com/marklogic/hub/hubcentral/migration/FlowMigrator.java

```diff
@@ -149,6 +149,10 @@ public void migrateFlows() {
             ObjectNode newSteps = nodeFactory.objectNode();
             for (Map.Entry<String, Step> entry : steps.entrySet()) {
                 Step step = entry.getValue();
+                if (step.getIsOnlyStepId()) {
```

Comment:
While I'd normally say it's good to add methods like this to Step, I want to get away from classes like Step, which are incomplete representations of JSON objects. For the purpose of FlowMigrator, just check the original ObjectNode to see if it only has one key, and if the name of that key is stepId.

Reply:
There was no way to get to the JSON after getLocalFlow() executed, so I had to save a reference to the JsonNode in Step. (and hid it from serialization with an attribute)